internal/nexodus/utils.go

```diff
@@ -46,6 +48,31 @@ func RunCommand(cmd ...string) (string, error) {
 	return string(output), nil
 }
 
+// RunCommandPipeline runs a pipeline of commands in a shell
+func RunCommandPipeline(commands ...string) (string, error) {
```

Comment:
I would prefer a different implementation for the pipeline that doesn't use `sh`.  See "Piping Between Processes" on this blog post for an example: https://www.dolthub.com/blog/2022-11-28-go-os-exec-patterns/

My reason is that it's less likely to introduce a security issue later. By using `sh` here, if anyone accidentally passes user-supplied input here that can be interpreted by `sh`, we have a shell-injection vulnerability. As a made-up example, imagine some new code uses an organization name as part of a command and also forgets to quote it properly. What happens if someone renames their organization `; rm -rf / ;` ?  😬 

I don't see any such chance with the code as-is, but changing the implementation would avoid the possibility in the future.

Reply:
Good callout, will fix it up!

Reply:
@russellb  have updated based on feedback. After looking through the PR i realise the scope has sort of creeped a bit more than the initial issue (changing variable naming, doing validation on routes for Darwin etc) which is a bit of a mistake... Happy to split these changes up into separate PRs if you'd be more comfortable with it, just let me know.

Reply:
Thanks for asking, but don't worry about it on this one. I'd rather that effort go toward something else. :)